handler/handler.go

```diff
@@ -51,26 +49,12 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, event *models.Cont
 		"keywords":      zebedeeData.Description.Keywords,
 		"keywordsLimit": keywordsLimit,
 	}
-	validKeywords, err := ValidateKeywords(zebedeeData.Description.Keywords, keywordsLimit)
-	if err != nil {
-		log.Info(ctx, "failed to get keywords limit", logData)
-		return err
-	}
 
 	//Mapping Json to Avro
-	searchData := models.SearchDataImport{
-		DataType:        zebedeeData.DataType,
-		JobID:           "",
-		SearchIndex:     "ONS",
-		CDID:            zebedeeData.Description.CDID,
-		DatasetID:       zebedeeData.Description.DatasetID,
-		Keywords:        validKeywords,
-		MetaDescription: zebedeeData.Description.MetaDescription,
-		Summary:         zebedeeData.Description.Summary,
-		ReleaseDate:     zebedeeData.Description.ReleaseDate,
-		Title:           zebedeeData.Description.Title,
-		TraceID:         traceID,
-	}
+	searchData := models.MapZebedeeDataToSearchDataImport(zebedeeData, keywordsLimit)
+	searchData.TraceID = traceID
+	searchData.JobID = ""
```

Comment:
any reason for not moving optional JobID in mapper

Reply:
JobId doesn't come from the zebedee data. Ie. it is not part of the data returned by the PublishedData endpoint. It is additional data provided by a different source. This method is for mapping items in the Zebedee page data to the event struct.